HousingFinanceInterimApi/V1/UseCase/MoveHousingBenefitFileUseCase.cs

```diff
@@ -172,10 +176,22 @@ private async Task<List<GoogleFileSettingDomain>> GetGoogleFileSetting(string la
 
         private string CalculateNewFileName(File file)
         {
-            var fileNameDateMatches = _academyFilePattern.Matches(file.Name);
-            var fileNameDate = DateTime.ParseExact(fileNameDateMatches[1].Value, "ddMMyyyy", null);
+            // Handle null creation date
+            if (file.CreatedTime == null)
```

Comment:
A thing also worth checking is is the `CreatedTime` a nullable field within the `File` class?
If it's not, then this `if` statement will never trigger. If it's not nullable, then it's likely it would default to `DateTime.MinValue` or something like that _(pretty much 1900s)_.

Reply:
If the field is nullable, then the `file.CreatedTime.HasValue` check may be more appropriate.

Reply:
It is nullable but by default (and in the current implementation in our usecase) it is set from a non-nullable field (see other comment chain)